Gems/AudioSystem/Code/Source/Editor/AudioResourceSelectors.cpp

```diff
@@ -67,10 +66,32 @@ namespace AudioControls
     }
 
     //-------------------------------------------------------------------------------------------//
-    REGISTER_RESOURCE_SELECTOR("AudioTrigger", AudioTriggerSelector, ":/AudioControlsEditor/Icons/Trigger_Icon.png");
```

Comment:
can you also delete that macro? I think there is just one usage in some other place

Reply:
I'm scrawling through the code right now to get a better scope of what we can delete.  There's a few more that are registered via this macro: "Sound" (ok to delete it), "Model", and "GeomCache".  I'm looking to see if any real UI are actually referencing any of these.

Reply:
thanks!